styles/FeaturedWork.module.css

```diff
@@ -0,0 +1,84 @@
+.featured-work {
+  margin-top: 2%;
+  margin-bottom: 2%;
+  display: flex;
+  border-bottom: #000000 solid 1px;
+  border-top: #000000 solid 1px;
+}
+
+.featured-work__left-container {
+  width: 33%;
+  margin-right: 2%;
+  margin-bottom: 2%;
+  margin-top: 2%;
+  border-right: 1px solid #000000;
+}
+
+.featured-work__left-container h1 {
+  font-weight: 500;
```

Comment:
Is there any global font-weight or font style can use here?

Reply:
Yes, but if I use font-weight, it will similar with prototype.